configure.ac

```diff
@@ -138,6 +138,14 @@ AM_CONDITIONAL([ENABLE_TCTI_MSSIM], [test "x$enable_tcti_mssim" != xno])
 AS_IF([test "x$enable_tcti_mssim" = "xyes"],
 	AC_DEFINE([TCTI_MSSIM],[1], [TCTI FOR MS SIMULATOR]))
 
+AC_ARG_ENABLE([tcti-swtpm],
+            [AS_HELP_STRING([--disable-tcti-swtpm],
+                            [don't build the tcti-swtpm module])],,
+            [enable_tcti_swtpm=yes])
+AM_CONDITIONAL([ENABLE_TCTI_SWTPM], [test "x$enable_tcti_swtpm" != xno])
+AS_IF([test "x$enable_tcti_swtpm" = "xyes"],
+	AC_DEFINE([TCTI_SWTPM],[1], [TCTI FOR SWTPM]))
```

Comment:
We need to check if swtpm is installed, or at least whether the required header file is present, otherwise configure can succeed, but compilation fails with
```
src/tss2-tcti/tcti-swtpm.h:13:10: fatal error: swtpm/tpm_ioctl.h: No such file or directory
```
Unfortunately swtpm doesn't include a pkg-config file, so we'll need
```
AC_CHECK_HEADER([swtpm/tpm_ioctl.h], , AC_MSG_ERROR([Missing header swtpm/tpm_ioctl.h: install swtpm.]))
```
probably directly here in the `AS_IF`?

Reply:
Yes, if we use the header, we should add a check. I'll with my next push. Thanks!